src/features/defi/providers/yearn/components/YearnManager/Deposit/components/Approve.tsx

```diff
@@ -157,6 +157,7 @@ export const Approve: React.FC<YearnApproveProps> = ({ onNext }) => {
       learnMoreLink='https://shapeshift.zendesk.com/hc/en-us/articles/360018501700'
       onCancel={() => onNext(DefiStep.Info)}
       onConfirm={handleApprove}
+      contractAddress={contractAddress}
```

Comment:
`contractAddress` is the address of the token being staked, not the address of the actual contract. What you're looking for is `ssRouterContractAddress` from `investor-yearn` - it is the same ShapeShift router contract for all Yearn tokens stakings